ADAL/src/broker/ios/ADBrokerHelper.m

```diff
@@ -257,7 +257,6 @@ + (void)saveToPasteBoard:(NSURL*) url
 {
     UIPasteboard *appPasteBoard = [UIPasteboard pasteboardWithName:@"WPJ"
                                                             create:YES];
-    appPasteBoard.persistent = YES;
```

Comment:
Setting persistence for named pasteboards deprecated in iOS 10+ [setPersistent](https://developer.apple.com/documentation/uikit/uipasteboard/1622096-setpersistent?language=objc)

saveToPasteBoard in IdentityCore(MSAL) also uses non-persistent pasteboard. This code block is only invoked when link to install broker is set prompting for first time installation of broker.

Reply:
Thanks for making this change. Long overdue :)